{{cookiecutter.project_name}}/contracts/NFT.vy

```diff
@@ -89,8 +92,19 @@ event ApprovalForAll:
     operator: indexed(address)
     approved: bool
 
+{%- if cookiecutter.royalties == 'y' %}
+# @dev This emits when a royalty fee is paid to the contract creator
+# @param fee price at which the owner as changed it
+# @param amount sold NFT
+# @param buyer of the NFT
+event RoyalityInfo:
+    fee: indexed(uint256)
+    transactionDate: indexed(uint256)
+    amount: uint256
+    buyer: address
+{%- endif %}
+
 owner: public(address)
-isMinter: public(HashMap[address, bool])
```

Comment:
Why is this removed?

Reply:
Undo this change please. If you want to propose this as a separate change, I would put it in a different PR